### PR TITLE
Remove python to models.py (error)

### PR DESCRIPTION
**Pull request by betdream**
Remove  `python`  to models.py  (error)

### CODE REVIEW

File: tr/django_models/README.md [hunk 1]
```diff
@@ -110,7 +110,6 @@ INSTALLED_APPS = (
 Şimdi `blog/models.py` dosyasını açalım ve içindeki her şeyi silip şu kodu yazalım:
 
 ```
-python
 from django.db import models
```
Comment (helenst) [inline tr/django_models/README.md]: The word python is meant to be there but should be on the end of the previous line so the syntax highlighter knows this is python.

so you will have

```
```python
```

**Comment by helenst** [comment]
Hi @betdream thank you for your contribution! If you can move that "python" up to the line with the backticks rather than removing it, then we can get this merged! :sparkling_heart:

**Comment by aniav** [comment]
Hi @betdream, what Helen was suggesting is moving python to the same line as the ``` signs, now there are no changes to the original code :)

**Comment by betdream** [comment]
Thanks, i was wrong :) no error

**Pull request closed without merging**